src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetApprenticeshipsFilterValues/GetApprenticeshipsFilterValuesQueryHandler.cs

```diff
@@ -67,21 +69,31 @@ public async Task<GetApprenticeshipsFilterValuesQueryResult> Handle(GetApprentic
             return queryResult;
         }
 
+        private async Task<List<string>> GetDistinctSectors(GetApprenticeshipsFilterValuesQuery request, CancellationToken cancellationToken)
+        {
+            var standardUIds = await _dbContext.Apprenticeships
+                .WithProviderOrEmployerId(request).Where(x => x.ProgrammeType == ProgrammeType.Standard).Select(x => x.StandardUId).Distinct().ToListAsync(cancellationToken);
+
+            var sectors = await _dbContext.Standards.Where(x => standardUIds.Contains(x.StandardUId)).Select(x => x.Route).Distinct().ToListAsync(cancellationToken);
```

Comment:
@YuvrajPatil-DFE just for my own understanding here, are we saying that `Sector` = `Route` - they are the same thing, and the term can be used interchangeably?